tests/lib/rules/one-var.js

```diff
@@ -581,6 +646,7 @@ ruleTester.run("one-var", rule, {
         },
         {
             code: "var a = 1; var b = 2; var x, y; for (var z of foo) {var c = 3, baz = z; for (var d in baz) {}}",
+            output: "var a = 1; var b = 2; var x, y; for (var z of foo) {var c = 3; var baz = z; for (var d in baz) {}}",
```

Comment:
Can you add some tests that use destructuring, and different types of declarations?

For example:

```js
var {foo} = 1, [bar] = 2;

// fixed to

var {foo} = 1; var [bar] = 2;
```

```js
const foo = 1, bar = 2;

// fixed to

const foo = 1; const bar = 2;
```

Reply:
✅